ksefcli/README.ksefcli.md

```
@@ -46,28 +46,31 @@ sudo mv ksefcli /usr/local/bin/
 
 Wyszukiwanie numeru KSeF dla faktury o konkretnym numerze:
+
 ```bash
 $ ksefcli SzukajFaktur -q -c ksefcli.yaml --from "-1week" --to "now" --invoiceNumber '0004/26' | jq -r '.Invoices[0].KsefNumber'
 12312312312-20260117-XXXXXXXXXXXX-5C
 ```
 
 Przesyłanie faktury z użyciem konkretnego profilu:
+
 ```bash
 $ ksefcli PrzeslijFaktury -c ksefcli.yaml -f d03900-001.xml  -a firma2
```

markdownlint-cli2 (0.21.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)